kratos/tests/auxiliar_files_for_python_unittest/mdpa_files/test_model_part_io_read.mdpa

```diff
@@ -84,7 +84,7 @@ End NodalData
 
 Begin SubModelPart Inlets
 	Begin SubModelPartData
-	IS_RESTARTED 1
+	IS_RESTARTED True

```

Comment:
can we have a check for each of them?
i.e. one for `1`, one for `True` and one for `true`?

(not sure if there are enough vars of type `bool` in the Core)